src/main/java/com/untamedears/ItemExchange/utility/BlockUtility.java

```diff
@@ -0,0 +1,111 @@
+package com.untamedears.ItemExchange.utility;
+
+import org.bukkit.Bukkit;
+import org.bukkit.Location;
+import org.bukkit.Material;
+import org.bukkit.block.*;
+import org.bukkit.inventory.*;
+
+import com.untamedears.ItemExchange.DeprecatedMethods;
+import com.untamedears.ItemExchange.ItemExchangePlugin;
+
+public class BlockUtility {
+
+	public static final BlockFace[] CardinalFaces = new BlockFace[] {
+			BlockFace.NORTH,
+			BlockFace.EAST,
+			BlockFace.SOUTH,
+			BlockFace.WEST,
+			BlockFace.DOWN,
+			BlockFace.UP
+	};
+
+	public static BlockFace GetFacingDirection(Block block) {
```

Comment:
Unneccesary, use https://hub.spigotmc.org/javadocs/spigot/org/bukkit/material/Button.html#getAttachedFace-- instead

Reply:
Also method names should all start with a lower case letter

Reply:
1. Tried to use the button class originally. View the conversation in Code-it-Yourself Club, #itemexchange, search for "I tried using Button's getAttachedFace() and Chest's getFacing(), but I wasn't getting the right data" and "Trying to do the button press with Bukkit's Button() class, but no luck, even when manually causing block updates.... back to binary trickery"

2. Done.